Bluewire.Conventions/BranchSemantics.cs

```diff
@@ -30,5 +31,25 @@ public string[] GetRemoteBranchFilters(params BranchType[] types)
         {
             return types.Distinct().Select(t => t.BranchFilter).Where(b => !String.IsNullOrWhiteSpace(b)).Select(b => $"*/{b}").ToArray();
         }
+
+
+        // Assumes we always create a tag when we start a new version
+        public Ref GetVersionZeroRef(SemanticVersion semVer)
+        {
+            return new Ref(semVer.Major + "." + semVer.Minor);
+        }
+
+        // Assumes sementics defined in BranchType.cs
+        public Ref GetVersionLatestRef(SemanticVersion semVer)
+        {
+            switch (semVer.SemanticTag)
+            {
+                case "beta": return new Ref("master");

```

Comment:
Can't use `master` if there exists an appropriate `maint/` tag. The caller needs to watch out for `master` as an end ref and do something special I think.